src/h5web/vis-packs/nexus/NexusPack.test.tsx

```diff
@@ -83,3 +84,76 @@ test('show error when encountering malformed NeXus metadata', async () => {
 
   expect(errorSpy).toHaveBeenCalledTimes(8); // React logs two stack traces per error
 });
+
+test('cancel and retry slow fetch of NxSpectrum', async () => {
+  jest.useFakeTimers();
+  await renderApp();
+
+  await selectExplorerNode('resilience/slow_nx_spectrum');
+  expect(await screen.findByText(/Loading data/)).toBeVisible();
+
+  const errorSpy = mockConsoleMethod('error');
+  userEvent.click(await screen.findByRole('button', { name: /Cancel/ }));
+
+  expect(await screen.findByText('Request cancelled')).toBeVisible();
+  expect(errorSpy).toHaveBeenCalledTimes(2); // React logs two stack traces
+
+  // Move dimension slider
```

Comment:
I think the comment could be more meaningful here to explain the purpose of this part. Something like `Request should be still cancelled after moving dimension slider`